src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (<p className="Person__age">I am {age}</p>)}
+
+      <p className="Person__partner">{isMarried ? `${partnerName} is my ${partnerStatus}` : `I am not married`}</p>
```

Comment:
```suggestion
      <p className="Person__partner">
        {isMarried 
          ? `${partnerName} is my ${partnerStatus}` 
          : `I am not married`
        }
      </p>
```

<img width="1086" alt="image" src="https://github.com/mate-academy/react_person/assets/84296041/339a76cd-3d8d-42b4-8999-c9ea2d9ec2d5">